src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -1,12 +1,88 @@
 package frc.team2412.robot.subsystem;
 
-import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
-
+import com.ctre.phoenix.motorcontrol.TalonFXControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
```

Comment:
The leads recommend use of the WPI_TalonFX library, not TalonFX.  Sorry, if there was a miscommunication.
The WPI lib includes support for liveWindow and other features which will likely be very handy for debugging and troubleshooting the synchronization of the climbing subsystem.